scripts/index.js

```diff
@@ -33,6 +33,23 @@ function squaresVille(flats) {
 
 squaresVille([2, 5, 6, 7, 99]);    
 
+//*************cities #1**************
+var cities = [
+    { name: 'Los Angeles', temperature: 60.0},
+    { name: 'Atlanta', temperature: 52.0 },
+    { name: 'Detroit', temperature: 48.0 },
+    { name: 'New York', temperature: 80.0 }
+];
+
+
+function coldTimeInTheCity(cities) {
+    let niceAndCold = cities.filter(val => {
+        return val < 70;
+    });
+    console.log(niceAndCold);
```

Comment:
Hey I noticed I couldn't get an output from this function. I think you're on the right track. Were you wanting to return all the cities with temperatures < than 70? What if we tried...
`function coldTimeInTheCity(cities) {
    let niceAndCold = cities.filter(val => {
        return val.temperature < 70;
    });
    return niceAndCold;
}
coldTimeInTheCity(cities);
`